### DISCUSSION TITLE
cross break

### DISCUSSION THREAD

**Discussion by JaapStruyk** [shaise/FreeCAD_SheetMetal#300] [Ideas]
Is there a way to add a cross break to a sheet?
![cross-break_panel-2-620x325](https://github.com/shaise/FreeCAD_SheetMetal/assets/5475283/1dcf0e9f-44bd-4fc3-9ee8-08a6dc051b12)

**Comment by shaise** [comment]
There is a punch function that might do it:
![image](https://github.com/shaise/FreeCAD_SheetMetal/assets/5813568/1946cb9d-6614-4146-94da-70cdf2a5a1b9)
But I'm not sure its appropriate for that.
Please try also the freecad forum: https://forum.freecad.org you might get more answers
